client/signup/main.jsx

```diff
@@ -644,7 +659,9 @@ export default connect(
 			siteDomains,
 			siteId,
 			siteType: getSiteType( state ),
-			shouldShowMockups: get( steps[ ownProps.stepName ], 'props.showSiteMockups', false ),
+			shouldShowMockups:
+				! getSiteTypePropertyValue( 'slug', this.props.siteType, 'hideSiteMockups' ) &&
```

Comment:
I guess this should be `ownProps.siteType`?

Reply:
Damn, bad rebase. Thanks for catching!